e2e/utils/command-utils.ts

```diff
@@ -120,8 +120,8 @@ export function getPackageManagerCommand({
   list: string;
   runLerna: string;
 } {
-  const npmMajorVersion = getNpmMajorVersion();
-  const yarnMajorVersion = getYarnMajorVersion(path);
+  const npmMajorVersion = getNpmMajorVersion() ?? -1;
+  const yarnMajorVersion = getYarnMajorVersion(path) ?? -1;
```

Comment:
tbh I just added the `?? -1` because I was not sure what's the best approach here. Like, maybe if no major version detected, do not return the `npm` object ?

Reply:
I don't think it matters, because if npm or yarn are not available, your install would fail if that package manager was detected based on the lock file.

Reply:
I would leave those two lines as they were

Reply:
I added a check below! :D